packages/republik/graphql/index.js

```diff
@@ -4,6 +4,5 @@ const { graphql: scalars } = require('@orbiting/backend-modules-scalars')
 const local = addTypes(loadModule(__dirname), [scalars])
 
 const { graphql: auth } = require('@orbiting/backend-modules-auth')
```

Comment:
One thing I noticed, is that we still require, but never use `const { graphql: auth } = require('@orbiting/backend-modules-auth')` in the following file: `servers/graphql/server.js`. I think that was forgotten when we merged the two servers.